std/format.d

```diff
@@ -4052,103 +4054,96 @@ here:
     const string[] widths = [ "", "0", "4", "20" ];
     const string[] precs = [ "", ".", ".0", ".4", ".20" ];
     const string formats = "sdoxXeEfFgGaA";
-  /+
```

Comment:
1. I have no idea.
2. It tests if formattedWrite behaves as snprintf for "sdoxXeEfFgGaA" (which it should)